core/src/com/po/fuck/model/Player.java

```diff
@@ -1,39 +1,42 @@
-package com.po.fuck;
+package com.po.fuck.model;
+
+import static com.po.fuck.model.Constants.DEFAULT_SPEED;
 
 import com.badlogic.gdx.Gdx;
 import com.badlogic.gdx.InputMultiplexer;
 import com.badlogic.gdx.graphics.Texture;
 import com.badlogic.gdx.graphics.g2d.Sprite;
 import com.badlogic.gdx.math.Vector2;
-import com.po.fuck.movement.BasicMovement;
-import com.po.fuck.movement.Boost;
-import com.po.fuck.weapons.LaserGun;
-import com.po.fuck.lifetime.Manager;
-import com.po.fuck.lifetime.Managed;
-
-import static com.po.fuck.Constants.DEFAULT_SPEED;
+import com.po.fuck.controller.KeyboardController;
+import com.po.fuck.controller.MouseController;
+import com.po.fuck.model.lifetime.Managed;
+import com.po.fuck.model.lifetime.Manager;
+import com.po.fuck.model.movement.BasicMovement;
+import com.po.fuck.model.movement.Boost;
+import com.po.fuck.model.weapons.LaserGun;
 
 /**
  * Class representing the player entity in the game.
  * Inherits from the Entity class and adds functionality specific to the player.
  */
 public class Player extends Entity {
     {
-        sprite = new Sprite(new Texture("FUCKerWithoutHands2.png"));
         weapon = Manager.create(new LaserGun(this));
         movement = Manager.create(
                 new Boost(
                         new BasicMovement(this, DEFAULT_SPEED),
                         Constants.BOOST_DISTANCE, Constants.BOOST_DURATION, Constants.BOOST_COOLDOWN));
         teamTag = Constants.PLAYER_TEAM_TAG;
+        width = new Sprite(new Texture("FUCKerWithoutHands2.png")).getWidth();
```

Comment:
If you want to merge it now, then these values should be passed through constructor (more preferred way) or as constants.

Reply:
Which values are talking about? All from the initializer block?

Reply:
width/height